cred_helpers.go

```diff
@@ -2,18 +2,18 @@ package lifecycle
 
 import (
 	"encoding/json"
+	"fmt"
 	"github.com/google/go-containerregistry/pkg/name"
 	"os"
 	"path/filepath"
 	"regexp"
 )
 
-func SetupCredHelpers(refs ...string) error {
-	dockerPath := filepath.Join(os.Getenv("HOME"), ".docker")
+func SetupCredHelpers(home string, refs ...string) error {
```

Comment:
Instead of coupling this to the idea of `$HOME`, maybe we inject the full path to `.docker` instead?

Reply:
That sounds good to me.